changelog/v1.4.0-beta1/sds_cluster.yaml

```
@@ -0,0 +1,4 @@
+changelog:
+  - type: NEW_FEATURE
+    description: Allow specifying an a target cluster in the SDS config.
```

grammar, plus needs more detail
```suggestion
    description: Allow specifying a target cluster in the SDS config. Istio 1.5 uses cluster for SDS, so this change allows us to support this use case.
```